pkg/oc/bootstrap/clusteradd/components/registry/registry_install.go

```diff
@@ -31,6 +34,24 @@ func (r *RegistryComponentOptions) Name() string {
 	return "openshift-image-registry"
 }
 
+// ensureRemoteRegistryStoragePermissions ensures the remote host directory for registry storage have write access permissions
+// so the registry can successfully write data into it.
+// TODO: This is a remote docker snowflake
+func (r *RegistryComponentOptions) ensureRemoteRegistryStoragePermissions(dir string, dockerClient dockerhelper.Interface) error {
+	glog.V(5).Infof("Ensuring the write permissions in remote directory %s", dir)
+	_, rc, err := run.NewRunHelper(dockerhelper.NewHelper(dockerClient)).New().
+		Image(r.InstallContext.ClientImage()).
+		DiscardContainer().
+		Privileged().
+		Bind(fmt.Sprintf("%s:/pv", dir)).
+		Entrypoint("/bin/bash").
+		Command("-c", "mkdir -p /pv/registry && chmod 0777 /pv/registry").Run()
```

Comment:
@mfojtik Even with this I am not able to see the changed file permission in the VM.

```$ ls -l /var/lib/origin/cluster-up/tmp/base/openshift.local.pv/
total 0
drwxr-xr-x. 3 root root 22 May  2 01:51 registry
```